source/Preferences.cpp

```diff
@@ -37,8 +37,11 @@ namespace {
 	const string EXPEND_AMMO = "Escorts expend ammo";
 	const string FRUGAL_ESCORTS = "Escorts use ammo frugally";
 
-	const vector<double> ZOOMS = {.25, .35, .50, .70, 1.00, 1.40, 2.00};
-	int zoomIndex = 4;
+	// This controls the range of zoom scales the player can switch to.
+	// larger values are close-up and small values are farther away
+	const vector<double> ZOOMS = {.105, .125, .149, .177, .210, .250, 0.297, 0.354, 0.420, 0.500, 0.595, 0.707, 0.841,
+		1.000, 1.189, 1.414, 1.682, 2.000, 2.378};
+	int zoomIndex = 11;
```

Comment:
This level of precision is unnecessary, and we don't need trailing zeroes here.
```suggestion
	const vector<double> ZOOMS = {.105, .125, .15, .175, .21, .25, 0.3, 0.35, 0.42, 0.5, 0.6, 0.7, 0.84,
		1., 1.2, 1.4, 1.68, 2., 2.4};
	int zoomIndex = 13;
```

Reply:
Sorry, I pushed the change to the max zoom before applying this. I have pushed a second commit that removes all the extraneous zeroes and changed the index to 13.

Thank you for the suggestion, though. Normally I apply these first.